challenges.tex

```diff
@@ -0,0 +1,530 @@
+	\node at (0,5) [circle,fill,inner sep=1.5pt,label={[font=\scriptsize]45:$(0,5)$}]{};
+	\draw[line width=2pt,red,->] (0,0) -- node[above left,pos=1] {$x$} (-1.5,0);
+	\draw[line width=2pt,red,->] (0,0) -- node[below left,pos=1] {$y$} (0,1.5);
+\end{tikzpicture}
+	\caption{The \SI{5 x 5}{\metre} playing field.}
+	\label{fig:challenge-field}
+\end{figure}
+
+The entire area belongs to a single team. The bottom left-most
+\SI{1 x 3}{\metre} area is called the insertion zone and provides the starting
+positions for up to three robots.
+Contrary to the full RCLL field, it is not necessary to build partial walls
+around the insertion zone and robots may be move freely in and out of the
+insertion zone.
+\subsection{Mockup Machines}\label{sec:machines}
+In case no real \ac{MPS} stations are available, replications
+(so called \emph{mockup machines}) may be used, that do not need to
+physically perform the respective production steps. Instead, that work may
+be carried out by a human supervisor (see \refsec{sec:operators}).
+The minimum requirements for a mockup machine are specified in the following.
+
+Mockup machines are required to have the same box-like base-shape as specified
+in the RCLL rulebook.
+Additionally, the following parts need to be mounted:
+\begin{itemize}
+	\item a model of the conveyor belt
+	\item a shelf on the front right side of the box on stations replicating a \ac{CS}
+	\item either a shelf, a slide or a conveyor belt on the front right side
+of the box, such that it is accessible from the front on stations replicating a \ac{RS}
+\end{itemize}
+Models for a conveyor belt, shelf and slides can be found in the RCLL rulebook
+repository\footnote{\url{https://github.com/robocup-logistics/rcll-rulebook/tree/common/challenges/mock_up_models}}.
+
+The building materials for the models must be opaque, but may have any color.
+
+In order to compete in all main challenges, a minimum number of 3 mockup
+machines are required. On higher difficulty some challenges may require
+4 mockup stations and some secondary challenges require 7 stations.
+
+\subsection{Remote Setup}
+In case a competition is carried out remotely, a proper local setup has to
+be established and approved by the OC.
+Requirements include a proper camera setup that covers the field sufficiently,
+such that external viewers can verify the integrity of each challenges,
+as well as an approval for every mockup machine and robot that is used.
+After registration the OC will verify the field setup in a video call.
+The validation call will be scheduled individually for each team
+to account for timezones.
+
+\section{Game Play}
+\subsection{Competition Scope}
+Each team gets slots assigned to participate in challenges.
+Each slot can be used to do at most one challenge and is split in two phases:
+1. In the \texttt{SETUP} phase the participants ensure that
+\begin{itemize}
+	\item the video stream is setup,
```

Comment:
Should we fix a software to use here or will it be provided by the RoboCup federation? Not that every team comes up with a different application to stream their games.

Reply:
I think we agreed on Zoom to record and stream.
And there will be a inspection of the recording setup of each team in the first week of June (once all teams are registered).